src/filters/FilterPrice/FilterPriceCustomStep.ts

```diff
@@ -0,0 +1,40 @@
+import { MarketFilterCustomStep } from '../../modules';
+import { FilterPrice, PriceValidation } from './index';
+
+export const countDecimals = value => {
+    if (Math.floor(value) === value) { return 0; }
+    const decimalPart = value.toString().split('.')[1];
```

Comment:
In case value is `null`, there will be an error

Reply:
added default parameter, thanks